checkout/access-checkout-android/src/main/java/com/worldpay/access/checkout/api/session/SessionResponseInfo.kt

```diff
@@ -3,6 +3,12 @@ package com.worldpay.access.checkout.api.session
 import com.worldpay.access.checkout.client.SessionType
 import java.io.Serializable
 
+/**
+ * [SessionResponseInfo] class to hold information from the Session Response
```

Comment:
this holds the actual response body as well as other metadata about the response

Reply:
updated